source-http-ingest/src/transactor.rs

```diff
@@ -30,13 +30,9 @@ impl Acknowledgements {
 pub struct Emitter(io::Stdout);
 
 impl Emitter {
-    pub async fn emit_doc(&mut self, binding: u32, doc: &impl Serialize) -> anyhow::Result<()> {
+    pub async fn emit_doc(&mut self, binding: u32, doc_json: String) -> anyhow::Result<()> {
         use tokio::io::AsyncWriteExt;
 
-        // We intentionally don't use `write_capture_response` here because we don't want to wait
-        // for the flush call to complete.
-        let doc_json = serde_json::to_string(doc).context("serializing output document")?;
-
         let resp = Response {
             captured: Some(Captured { binding, doc_json }),
```

Comment:
There's a gotcha here: if the POST body contains newlines, we'll plumb them through and break the newline delimited protocol.

`models::RawValue` takes care of this case to ensure that newlines are removed.

Reply:
Good catch! I just switched to `models::RawValue`.